documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -523,6 +504,77 @@ Add a `WEB-INF/jboss-deployment-structure.xml` in your archive with the followin
 More information about the descriptor can be found in the
 https://docs.jboss.org/author/display/WFLY10/Class+Loading+in+WildFly[WildFly documentation].
 
+===== Server provisioning via other build tools
+
+The provisioning tools have been developed mainly for Maven users, but there are options for users of other build tools too.
+
+Gradle users can use the link:https://plugins.gradle.org/plugin/org.wildfly.build.provision[org.wildfly.build.provision] plugin, available on the official Gradle plugin portal.
+
+Both the Maven and the Gradle plugins are just thin wrappers invoking a standalone provisioning library, so you could also integrate this library with any other JVM based tool.
+
+See also https://github.com/wildfly/wildfly-build-tools[WildFly provisioning build tools].
+
+[[hibernate-ogm-jboss-modules-feature-packs]]
+==== List of the Hibernate OGM WildFly/JBoss feature packs
+
+===== Core feature pack
+
+This is the base feature pack, extended by all Hibernate OGM dialect feature packs.
+It includes the main module:
+
+* _org.hibernate.ogm_, containing Hibernate OGM core module
+
+It includes Hibernate ORM {hibernate-orm-version} modules, required by Hibernate OGM {hibernate-ogm-version}.
+Several more modules are included but you should not need to refer to the other ones directly.
```

Comment:
I don't find this sentence very clear, which other modules. why mention them if the user doesn't have to refer to them.

Reply:
We can remove it, I copied from Search docs, but I agree with you is useless.